VERSION

```diff
@@ -1 +1 @@
-1.16.0
\ No newline at end of file
+1.16.1
```

Comment:
It looks like you introduced a trailing newline here. Did you use `scripts/update-version` to update the version?

Reply:
I didn't know that we have a script for this, just updated.